metagraph/src/cli/query.cpp

```diff
@@ -85,6 +86,26 @@ std::string QueryExecutor::execute_query(const std::string &seq_name,
 
         output += '\n';
 
+    } else if (query_coords) {
+        auto result = anno_graph.get_kmer_coordinates(sequence,
+                                                      num_top_labels,
+                                                      discovery_fraction);
+
+        if (!result.size() && suppress_unlabeled)
+            return "";
+
+        output += seq_name;
+
+        for (const auto &[label, tuples] : result) {
+            output += "\t<" + label + ">";
+            for (const auto &coords : tuples) {
+                output += ":";
+                output += fmt::format("{}", fmt::join(coords, ","));
+            }
+        }
```

Comment:
This produces a lot of unnecessary output for sequences with long matching stretches. So how about reporting something like `<chrX>:1-2:5-6` (with inclusive ranges similar to what `samtools` does)

Reply:
I suggest we use this for now and change it if we see that it's objectively better or others request this.

Reply:
I was suggesting this to make the output more human readable, not so much for performance reasons.

Reply:
I think the ease of parsing is more important. Not clear which is easier to parse, though. I suggest we keep things as is and think about this and change this when we collect more data.